ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/LedgerTestCasesRunner.scala

```diff
@@ -208,10 +214,10 @@ final class LedgerTestCasesRunner(
             )(materializer, materializer.executionContext)
           } yield concurrentTestResults ++ sequentialTestResults
 
-        testResults
-          .recover { case NonFatal(e) =>
+        testResults.recover {
```

Comment:
You may avoid having to resort to `isInstanceOf` by pattern matching on and rethrowing the `Errors.FrameworkException` first.

Reply:
Yeah, I tried that but it seemed uglier to me. I don't really mind either way though.